src/libs/actions/ReimbursementAccount/store.ts

```diff
@@ -1,39 +1,41 @@
-import lodashGet from 'lodash/get';
+import type {OnyxEntry} from 'react-native-onyx';
 import Onyx from 'react-native-onyx';
-import _ from 'underscore';
 import BankAccount from '@libs/models/BankAccount';
 import ONYXKEYS from '@src/ONYXKEYS';
+import type * as OnyxTypes from '@src/types/onyx';
+import type {ACHData} from '@src/types/onyx/ReimbursementAccount';
+import type {EmptyObject} from '@src/types/utils/EmptyObject';
 
 /** Reimbursement account actively being set up */
-let reimbursementAccountInSetup = {};
+let reimbursementAccountInSetup: ACHData | EmptyObject = {};
```

Comment:
Maybe we can do the same you did with `credentials` in this file, wdyt? But also I haven't found this values are really being used. It's used only in a function which is used nowhere...
```suggestion
let reimbursementAccountInSetup: OnyxEntry<ACHData> = null;
```

Reply:
Hmm but now I think about it and I'm not sure if we use `OnyxEntry` for keys of `Onyx` types.
I mean that `ACHData` is not `OnyxEntry` itself but it is a key of `ReimbursementAccount` 🤔

Reply:
I'm okay to leave it as is then 👌